README.md

```diff
@@ -31,250 +31,223 @@ Same browser support as [Vue.js 2](https://github.com/vuejs/vue/blob/dev/README.
+|Error|Object|@error|Triggered when an error occurred.|
 
 ### Public methods
-
-#### print(dpi, pageList) * _experimental_ *
-  * `dpi`: the print rezolution of the document (try 100).
-  * `pageList`: the list (array) of pages to print.
+|Name|Parameters|Description|
+|----|----------|-----------|
+|print|dpi, pageList|  **_experimental_** <br/> `dpi`: the print rezolution of the document (try 100).<br/> `pageList`: the list (array) of pages to print.|
 
 ### Public static methods
-
-#### createLoadingTask(src)
-  * `src`: see `:src` prop  
-  This function creates a PDFJS loading task that can be used and reused as `:src` property.  
-  The loading task is a promise that resolves with the PDFJS pdf document that exposes the `numPages` property (see example below).
-
+|Name|Parameters|Description|
+|----|----------|-----------|
+|createLoadingTask|src|`src`: see `:src` prop This function creates a PDFJS loading task that can be used and reused as `:src` property.<br/>The loading task is a promise that resolves with the PDFJS pdf document that exposes the `numPages` property (see example below).|
 
 ## Examples
 
 ##### Example - current page / page count
-```
+```js
 <template>
-	<div>
-		{{currentPage}} / {{pageCount}}
-		<pdf
-			src="https://cdn.mozilla.net/pdfjs/tracemonkey.pdf"
-			@num-pages="pageCount = $event"
-			@page-loaded="currentPage = $event"
-		></pdf>
-	</div>
+  <div>
+    {{currentPage}} / {{pageCount}}
+    <vue-pdf
+      src="https://cdn.mozilla.net/pdfjs/tracemonkey.pdf"
+      @num-pages="pageCount = $event"
+      @page-loaded="currentPage = $event" />
+  </div>
 </template>
 
 <script>
-
-import pdf from 'vue-pdf'
-
+import VuePdf from 'vue-pdf'
 export default {
-	components: {
-		pdf
-	},
-	data() {
-		return {
-			currentPage: 0,
-			pageCount: 0,
-		}
-	}
-}
-
+  data() {
+    return {
+      currentPage: 0,
+      pageCount: 0
+    }
+   },
+   components: {
+    VuePdf
+  }
 </script>
 ```
 
-
 ##### Example - display multiple pages of the same pdf document
-```
+```js
 <template>
-	<div>
-		<pdf
-			v-for="i in numPages"
-			:key="i"
-			:src="src"
-			:page="i"
-			style="display: inline-block; width: 25%"
-		></pdf>
-	</div>
+  <div>
+    <vue-pdf
+      v-for="i in numPages"
+      :key="i"
+      :src="src"
+      :page="i"
+      style="display: inline-block; width: 25%" />
+  </div>
 </template>
 
 <script>
-
-import pdf from 'vue-pdf'
-
+import VuePdf from 'vue-pdf'
 var loadingTask = pdf.createLoadingTask('https://cdn.mozilla.net/pdfjs/tracemonkey.pdf');
-
 export default {
-	components: {
-		pdf
-	},
-	data() {
-		return {
-			src: loadingTask,
-			numPages: undefined,
-		}
-	},
-	mounted() {
-
-		this.src.then(pdf => {
-
-			this.numPages = pdf.numPages;
-		});
-	}
+  data() {
+    return {
+      src: loadingTask,
+      numPages: undefined,
+    }
+  },
+  mounted() {
+    this.src.then(pdf => {
+      this.numPages = pdf.numPages;
+   });
+  },
+  components: {
```

Comment:
please move "components" at the top (https://vuejs.org/v2/style-guide/#Component-instance-options-order-recommended)